.env

```diff
@@ -2,3 +2,4 @@ PORT=5000
 OPENAI_API_KEY="get in from OPEN AI"
 JWT_ACCESS_SECRET="ImagineAiAccessSecret"
 JWT_REFRESH_TOKEN="ImagineAiRefreshToken"
+MONGODB_URI=mongodb+srv://imagineAI:imagineAI123@cluster0.gqzse4b.mongodb.net/
```

Comment:
Refrain from adding the actual credentials here.
This would expose the credentials.

PS: Whoever is maintaining the above cluster, please rotate the password.
Author please make sure that the password of the exposed cluster is changed.